ui/razercfg

```diff
@@ -387,6 +414,7 @@ def usage():
 	print("-R|--getres                         Prints the resolutions")
 	print("-f|--freq [PROF:]FREQ               Changes the scan frequency")
 	print("-F|--getfreq                        Prints the frequencies")
+	print("-n|--mapping [PROF:]RES[xRES]       Changes the resolution of the currently used mapping")
```

Comment:
This way we can only change the current mapping.
It would be good to have a way to change arbitrary mappings. Even those that are not active. Maybe via an optional mapping index?